database.py

```diff
@@ -0,0 +1,26 @@
+import psycopg2
+class DatabaseConnection:
+    def __init__(self):
+        try:
+            self.connection = psycopg2.connect(
+                "dbname='maintenence-tracker' user='admin' password='admin' host='localhost' port='5432' ")
+            self.connection.autocommit = True
+            self.cursor = self.connection.cursor()
+        except:
+            return "could not connect to the database"
+
+    def create_table_users(self):
+        create_database_table = "CREATE TABLE users(id SERIAL PRIMARY KEY, name VARCHAR(100) NOT NULL, email VARCHAR(100) NOT NULL, username VARCHAR(50) NOT NULL, password VARCHAR(100) NOT NULL, create_date TIMESTAMP DEFAULT  CURRENT_TIMESTAMP, is_admin BOOLEAN)"
```

Comment:
consider changing this line to 
```python
create_database_table = "CREATE TABLE IF NOT EXISTS users(id SERIAL PRIMARY KEY, name VARCHAR(100) NOT NULL, email VARCHAR(100) NOT NULL, username VARCHAR(50) NOT NULL, password VARCHAR(100) NOT NULL, create_date TIMESTAMP DEFAULT  CURRENT_TIMESTAMP, is_admin BOOLEAN)"
```
This will skip table creation if the table already exists